subsys/bluetooth/Kconfig

```diff
@@ -164,6 +164,14 @@ config BT_DATA_LEN_UPDATE
 	depends on !BT_CTLR || BT_CTLR_DATA_LEN_UPDATE_SUPPORT
 	help
 	  Enable support for Bluetooth v4.2 LE Data Length Update procedure.
+
+config BT_PER_ADV_SYNC_TRANSFER_RECEIVER
```

Comment:
Could we shorten this a bit, eg `BT_PAST_XFER_RECV`? Same with the other option.

Reply:
`BT_PER_ADV_SYNC_TRANSFER_RECEIVER` does better match the `bt_le_per_adv_sync_transfer` API though. 

Btw, the `t` in `past` is for transfer, so it could just be `BT_PAST_RECEIVER` and `BT_PAST_SENDER` if that works for you

Reply:
Ah , I missed the existing APIs. They have quite long names, but I suppose it makes sense to stay consistent. So either rename the API or be consistent with the KConfig symbols.

Reply:
Indeed. I remember when choosing the long `bt_le_per_adv_sync_transfer` name, was that we already had the `bt_le_per_adv_sync` API, based on the `bt_le_per_adv` API which in turn is based on the `bt_le_ext_adv` API. 

So at that time it did make sense to use the longer and more explicit name to be consistent with the naming of the other APIs.

Reply:
Will leave as is unless there is a major objection